### PR TITLE
Maillist

### PR DESCRIPTION
**Pull request by shanchenqi**

### CODE REVIEW

File: dags/oss_know/libs/base_dict/variable_key.py [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 NEED_INIT_GITS = "need_init_gits"
 
+
```
Comment (crystaldust) [inline dags/oss_know/libs/base_dict/variable_key.py:15]: Remove the unnecessary newlines(keep one newline to split different logical blocks is enough)

File: dags/oss_know/libs/maillist/archive.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+
+class EmailArchive:
+    def __init__(self, project_name, list_name, url_prefix, start_since=None):
+        self.project_name = project_name
+        self.dirpath = path.join(project_name, list_name)  # TODO Is this proper?(dirpath should just be temp)
+        self.url_prefix = url_prefix
+        self.start_since = start_since
+        self.list_name = list_name
+
```
Comment (crystaldust) [inline dags/oss_know/libs/maillist/archive.py:32]: Keep one newline for methods, according to the PEP style guide.

File: dags/oss_know/oss_know_dags/dags_maillist/sync_maillist.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.models import Variable
+from oss_know.libs.base_dict.variable_key import OPENSEARCH_CONN_DATA, GITHUB_TOKENS, NEED_INIT_GITHUB_COMMITS_REPOS, MAIL_LISTS
+
+# irflow.providers.postgres.hooks.postgres
+# v0.0.1 初始化实现
```
Comment (crystaldust) [inline dags/oss_know/oss_know_dags/dags_maillist/sync_maillist.py]: Update the comments to describe maillist features.

File: dags/oss_know/oss_know_dags/dags_maillist/sync_maillist.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+
+    need_do_inti_sync_ops = []
+
+
+    mail_lists = Variable.get(MAIL_LISTS, deserialize_json=True)
+
+    for mail_list in mail_lists:
+        task_name = mail_list["project_name"]
+        i = 0
```
Comment (crystaldust) [inline dags/oss_know/oss_know_dags/dags_maillist/sync_maillist.py]: The assignment to i is no longer necessary here.

File: dags/oss_know/libs/base_dict/variable_key.py (hunk 1 above)
Comment (shanchenqi) [reply dags/oss_know/libs/base_dict/variable_key.py:15]: Done!

**Comment by shanchenqi** [comment]
I have pushed a new commit, could you please review it  again?@crystaldust

**Pull request closed without merging**